hw/top_earlgrey/data/ip/chip_aes_testplan.hjson

```diff
@@ -0,0 +1,234 @@
+                -Clear all registers by setting KEY_IV_DATA_IN_CLEAR and DATA_OUT_CLEAR in TRIGGER
+                 to 1.
+            -Check the output for correctness against the expected value.
+
+            -The above procedure should be perforemed twice. First for encryption. Second for
+            decryption of the result using the same key (and same IV if needed).
+            Finally it should be checked that the decryption result matches the original plaintext.
+            '''
+      features: [
+        "AES.MODE.ECB",
+        "AES.MODE.CBC",
+        "AES.MODE.CFB-128",
+        "AES.MODE.OFB",
+        "AES.MODE.CTR",
+      ]
+      stage: V2
+      si_stage: SV2
+      lc_states: ["TEST_UNLOCKED", "DEV", "PROD", "PROD_END", "RMA"]
+      tests: []
+    }
+    {
+      name: chip_sw_aes_interrupt_ecryption
+      desc: '''Verify that a multi-block encryption can be interrupted and continued.
+
+            This test should:
+              - start the encryption of a four-block message.
+              - interrupt it after two blocks by reading back the IV register.
+              - perform an encryption of a different message using a different key.
+              - re-store and complete the original encryption
+
+            Note: AES should start this test using the automatic (default) operation. This means
+                  that the MANUAL_OPERATION bit in CTRL_SHADOWED register is set to 0.
+
+            Procedure:
+            -Start the encryption of a four-block message following the procedure listed in
+             chip_sw_aes_multi_block.
+             Note: Don't write the third block into the DATA_IN.
+            -After encrypting two blocks, read the DATA_OUT and IV. Note that AES has
+             automatically updated the content of this register.
+            -De-initialize automatic operation, as described in chip_sw_aes_multi_block test.
+             This step is necessary to clear all registers.
+            -Perform an encryption using a different data and key following the procedure in
+             chip_sw_aes_multi_block. Message size, key size and mode can be chosen arbitrarily.
+            -Continue original encryption:
+                -Set AES to automatic operation (set MANUAL_OPERATION bit in CTRL_SHADOWED to 0)
+                -Set the mode by writing to CTRL_SHADOWED.MODE
+                -Write CTRL_SHADOWED.KEY_LEN to set the key length.
+                -Write to CTRL_SHADOWED.OPERATION to select encryption.
+                -Write the original key to KEY_SHARE registers.
+                -Write the previously-read value to IV register.
+                -Write the third message block to DATA_IN. The encryption should continue
+                 automatically, after these registers are written.
+                -Complete encryption following the steps from chip_sw_aes_multi_block procedure.
+            -Check that the encryption result matches the original plaintext.
+            '''
```

Comment:
This description is clear but can be simplified a bit (see previous comments).

Reply:
This is now better. Thanks!

Reply:
fixed